### PR TITLE
Plat 333 gem publish

### PR DESCRIPTION
**Pull request by TruptiHosmani**
Publish distributed_resque_worker to rubygems

            - bumping the version for publishing the gem

    Jira- https://yourmechanic.atlassian.net/browse/PLAT-333

### CODE REVIEW

File: lib/distributed_resque_worker.rb [hunk 1]
```diff
@@ -13,9 +13,20 @@ module DistributedResqueWorker
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Layout/LineLength
 
+
+  module ResqueFailure
```
Comment (siddyg) [inline lib/distributed_resque_worker.rb:13]: @TruptiHosmani do we want this in this gem? Shouldnt this be in the user of the gem instead?
Also do we want to introduce bugsnag dependency in this gem?

**Review by siddyg — CHANGES_REQUESTED** [review]

File: lib/distributed_resque_worker.rb (hunk 1 above)
Comment (TruptiHosmani) [reply lib/distributed_resque_worker.rb:13]: removed the rubocop lines and also bungsnag for this one.

File: Gemfile.lock [hunk 2]
```diff
@@ -3,6 +3,7 @@ PATH
   specs:
     distributed_resque_worker (0.1.0)
       aws-sdk (= 1.11.1)
+      bugsnag
```
Comment (sachinsaxena1996) [inline Gemfile.lock]: @TruptiHosmani if bugsnag is removed why its appearing in the lock file I think you need to run bundle and then push buth Gemfile and Gemfile.lock

File: .rubocop.yml [hunk 3]
```diff
@@ -9,4 +9,10 @@ Style/NumericLiterals:
 Metrics/MethodLength:
   Max: 100
 Metrics/BlockLength:
-  Max: 30
\ No newline at end of file
+  Max: 30
+Layout/LineLength:
```
Comment (sachinsaxena1996) [inline .rubocop.yml:13]: @TruptiHosmani BlockLength could be max 10 also the LineLength could be max 80

**Review by sachinsaxena1996 — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (TruptiHosmani) [reply Gemfile.lock]: Ah yes, will do that

File: .rubocop.yml (hunk 3 above)
Comment (TruptiHosmani) [reply .rubocop.yml:13]: ok, will update these

**Review by sachinsaxena1996 — APPROVED** [review]

**Review by siddyg — APPROVED** [review]

**Pull request merged**